Assets/FullSerializer/Source/Converters/fsDateConverter.cs

```diff
@@ -27,31 +27,45 @@ public class fsDateConverter : fsConverter {
         public override fsResult TrySerialize(object instance, out fsData serialized, Type storageType) {
             if (instance is DateTime) {
                 var dateTime = (DateTime)instance;
-                serialized = new fsData(dateTime.ToString(DateTimeFormatString));
+                if(Serializer.Config.SerializeDateTimeAsInteger) {
+                    serialized = new fsData(dateTime.Ticks);
+                } else {
+                    serialized = new fsData(dateTime.ToString(DateTimeFormatString));
+                    return fsResult.Success;
+                }
                 return fsResult.Success;
             }
 
             if (instance is DateTimeOffset) {
                 var dateTimeOffset = (DateTimeOffset)instance;
-                serialized = new fsData(dateTimeOffset.ToString(DateTimeOffsetFormatString));
+                serialized = new fsData(dateTimeOffset.Ticks);
```

Comment:
Hmm, I honestly can't remember - I wrote this ages ago and forgot to submit a PR... it looks like I thought I had a reason for sticking to string for `DateTimeOffset`. Maybe this code is just a left-over experiment? I'll have a look.